src/Scripting/VisualBasicTest/CommandLineRunnerTests.vb

```diff
@@ -160,27 +146,33 @@ logoText + "
 > ? System.Math.PI
 3,1415926535897931
 >", runner.Console.Out.ToString())
+            Finally
+                CultureInfo.DefaultThreadCurrentCulture = currentCulture
+                CultureInfo.DefaultThreadCurrentUICulture = currentUICulture
+            End Try
         End Sub
 
         <Fact()>
         <WorkItem(7133, "https://github.com/dotnet/roslyn/issues/7133")>
         Public Sub TestDisplayResultsWithCurrentUICulture2()
-            ' Tests that DefaultThreadCurrentUICulture is respected and not DefaultThreadCurrentCulture.
-            Dim logoText = String.Format(VBScriptingResources.LogoLine1, s_compilerVersion) + vbNewLine +
-VBScriptingResources.LogoLine2 + "
-
-" + ScriptingResources.HelpPrompt
-            Dim runner = CreateRunner(args:={}, input:="Imports System.Globalization
+            ' Save the current thread culture as it is changed in the test.
```

Comment:
:bulb: The `UseCulture` attribute should restore the culture at the end of the test. Is it possible to use it here?

Reply:
There are two differences.

1. UseCulture sets *Current thread.Culture* while this test sets *CultureInfo.DefaultThreadCulture*.
2. UseCulture also sets the culture at the beginning of the test

I think it is better the way it is but haven't tried UseCulture.